src/main/java/com/owncloud/android/lib/resources/notifications/RegisterAccountDeviceForProxyOperation.java

```diff
@@ -62,10 +64,9 @@ public RegisterAccountDeviceForProxyOperation(String proxyUrl, String pushToken,
         this.userPublicKey = userPublicKey;
     }
 
-    @Override
-    protected RemoteOperationResult run(OwnCloudClient client) {
-        RemoteOperationResult result = null;
-        int status = -1;
+    public RemoteOperationResult run() {
+        RemoteOperationResult result;
+        int status;
```

Comment:
same here, now `0` instead of `-1`

Reply:
It is always set: 
https://github.com/nextcloud/android-library/blob/9b552a4947cd88f7fdb90fa8ccdebcab6b084dda/src/main/java/com/owncloud/android/lib/resources/notifications/RegisterAccountDeviceForProxyOperation.java#L83-L86

(except if there is an error before, but then status is never read)